### PR TITLE
Service ports stab 1

### PR DESCRIPTION
**Pull request by n1ywb**
add --service-ports to run command opts and add run opts to test command

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -281,9 +282,10 @@ fn run(matches: &clap::ArgMatches) -> Result<()> {
         }
         "test" => {
             warn_if_pods_are_enabled_but_not_running(&proj)?;
-            let service = sc_matches.value_of("SERVICE").unwrap();
+            let opts = sc_matches.to_run_options();
             let cmd = sc_matches.to_exec_command();
-            proj.test(&runner, service, cmd.as_ref())?;
+            let service = sc_matches.value_of("SERVICE").unwrap();
```
Comment (emk) [inline src/main.rs]: Is there a reason why we want to extract `to_exec_command` before we extract `SERVICE`? If I recall correctly, `SERVICE` comes before the exec command.

Comment (n1ywb) [reply src/main.rs]: I did it for (possibly foolish) consistency w the run command https://github.com/faradayio/cage/blob/efe2b075129fcc7a677a700531c617684b323ab1/src/main.rs#L256

Comment (n1ywb) [reply src/main.rs]: Is the order even significant? These looked declarative to me but I could be wrong.

Comment (n1ywb) [reply src/main.rs]: @emk ping

**Comment by emk** [comment]
@n1ywb Ping! Do you want to revisit this, now that `cage` is being more actively maintained again?

**Pull request closed without merging**